components/list/test/list.test.js

```diff
@@ -258,6 +258,30 @@ describe('d2l-list-item', () => {
 
 	});
 
+	describe('property validation', () => {
+
+		it('should set color as expected when valid 6 character hex color', async() => {
```

Comment:
My hope is that the color validation can be tested as a helper, and then we only just need some visual-diffs to test that the color indicators shows the color in the correct place.